### ISSUE TITLE
Annoying SMS notifications when a messages have been received

### ISSUE THREAD

**Issue by gpoo** [WhisperSystems/Signal-Android#4518]
I see two different behaviours when I am in Canada and in Chile, whenever I send a SMS (non-secure text message).

In Canada (koodo), I see a check next to the message I sent, and that is. Good.
In Chile (virgin, a virtual provider), I see a check, but I also receive from 1 to 3 extra messages saying:

> Mensaje fue recibido por +56######### .

(which translates to `Message was received by +56######### .`)

I believe I receive 1 message for as many cell-operators (roaming) the message have passed to get to its destination.  And I guess that because the message is in Spanish, Signal does not parse it in order to hide it or change it for something else.

The annoying part is that in a conversation, it appears that infamous message every time I sent something, so the thread is hard to follow. Besides that I get a notification when I have not received an actual reply, tricking me.

If you point me in the right direction, I might provide a patch to fix it.

Update: Android 4.4, Signal 3.3.1 (when it just became Signal from TextSecure).

**Comment by moxie0** [comment]
We don't have any control over how carriers provide delivery receipts. If you don't like the way they're doing it, I think all you can do is disable delivery SMS delivery receipts (in Signal settings).

**Comment by gpoo** [comment]
I thought it would be a matter of parsing the received message. It looks simple to implement.

**Comment by gpoo** [comment]
FWIW, I have "SMS delivery reports" disabled, so it does not look like a solution.
